src/tactic/rename.lean

```diff
@@ -0,0 +1,111 @@
+/-
+Copyright (c) 2020 Jannis Limperg. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Jannis Limperg
+-/
+import tactic.core
+
+
+/-!
+# Better `rename` tactic
+
+This module defines a variant of the standard `rename` tactic, with the
+following improvements:
+
+* You can rename multiple hypotheses at the same time.
+* Renaming a hypothesis does not change its location in the context.
+-/
+
+
+open lean lean.parser interactive native
+
+
+namespace tactic
+
+private def drop_until_inclusive {α} [decidable_eq α] (a : α) : list α → list α
```

Comment:
With a different name, this could be public and moved to a list file. `list.after`?

Reply:
I've moved this to `list.after` (in `data/list/defs`).